Yank/commands/script.py

```diff
@@ -25,14 +25,17 @@
 YANK script
 
 Usage:
-  yank script (-y=FILEPATH | --yaml=FILEPATH)
+  yank script (-y=FILEPATH | --yaml=FILEPATH) [--platform=PLATFORM]
```

Comment:
I thought we decided in #441 to specify precision in the YAML file and the platform as part of the command. I can definitely put both of them in the YAML file (in fact it's going to make the implementation easier) if we changed our minds on this.